examples/workflows/cockroach_basic_cluster/main.tf

```diff
@@ -0,0 +1,87 @@
+variable "cluster_name" {
+  type     = string
+  nullable = false
+}
+
+variable "sql_user_name" {
+  type     = string
+  nullable = false
+  default  = "maxroach"
+}
+
+# Remember that even variables marked sensitive will show up
+# in the Terraform state file. Always follow best practices
+# when managing sensitive info.
+# https://developer.hashicorp.com/terraform/tutorials/configuration-language/sensitive-variables#sensitive-values-in-state
+variable "sql_user_password" {
+  type      = string
+  nullable  = false
+  sensitive = true
+}
+
+variable "cloud_provider" {
+  type     = string
+  nullable = false
+  default  = "GCP"
+}
+
+variable "cloud_provider_regions" {
+  type     = list(string)
+  nullable = false
+  default  = ["us-central1"]
+}
+
+terraform {
+  required_providers {
+    cockroach = {
+      source = "cockroachdb/cockroach"
+    }
+  }
+}
+provider "cockroach" {
+  # export COCKROACH_API_KEY with the cockroach cloud API Key
```

Comment:
Will they know what to do here?

Reply:
This is how the other examples work. I think it'd be a different change to improve this, as all the examples that have this comment should be updated in the same way. If you have ideas on how to reword this or to point to docs that explain this better, I think it'd be a good improvement.